### ISSUE TITLE
You are amazing man:) reloaded is great

### ISSUE THREAD

**Issue by GoogleCodeExporter** [kexplo/imame4all#124]
```
What steps will reproduce the problem?
1. MAME with new drivers works very nice on samsung galaxy S people !!!!
2.Thank You very much !!!! 


This is best emulating news for android , its as good for me as cracking cs3 
encryption news :) Its great, i tested battletoads and violent storm and they 
run smooth like on PC.
I wasnt so excited from long time.
```

Original issue reported on code.google.com by `2black...@gmail.com` on 28 Feb 2012 at 12:48

**Comment by GoogleCodeExporter** [comment]
```
I got all beatem ups from 90's working nicely, street fighter 2 and cps1 cps2 
games work good as well, some games like MK1/2/3 or Street Fighter Movie do lag 
but that doesnt bother me cause other games that i like work very well.
I would like to know how long it took you to port this version, i think best 
version to port would be the ones used by kaillera , there are 2 of them , one 
plays MK games smooth even on old 700mhz pc's and it isnt Mame32 v0.67 cause 
this one have not enough drivers and i remember playing some games on kaillera 
which arent available on this build, i think it was 0.72 version and after it 
there was massive slowdown and big CPU usage.So porting this one would be best 
option for people wanting MK games but for CPS3 games they would have to buy 2 
core phone or wait for finalburn port, i dont know but it looks like most 
people want to play MK games on MAME so 0.72 build would be good choice for 
them but i bet it took you long time to port this version it works great :) 
Basically there wasnt much known great games added since 0.72 till 118 and then 
there was CPS3.Great work, now i have to buy gamepads.
```

Original comment by `2black...@gmail.com` on 28 Feb 2012 at 1:48

**Comment by GoogleCodeExporter** [comment]
```
I wanted to port a mame version with the new rendering system becose I like new 
artwork stuff and I want to support it. As you say this is not a straight port. 
I have written a mame osd layer that works with the new rendering system. It is 
thread use double core CPUs.  Sdl it was not an option. I tried to downgrade to 
0.119 one of the version used by killera. The speed was equal. So I keep a mame 
previous  to c++ rework. I choose 0.134 becouse is the one used on wii Port so 
rom set are well know. 

0.72 was an option but I wanted to port a recent mame build as I told you 
before. 
```

Original comment by `seleuco....@gmail.com` on 28 Feb 2012 at 2:47

**Comment by GoogleCodeExporter** [comment]
```
Yes but as you know you can do very little to speed up games in MAME and people 
can only buy better hardware to get more fps in games.Speed in mame above 0.72 
versions is very similar.Is it possible to downgrade to 0.72 ? I dont have 
dualcore phone, galaxy S has 1ghz but if i would have galaxy S2(but that one 
costs more than my laptop) then emu probably would run very smooth on it, i am 
shocked that you ported such new version to android, i thought no one would do 
that.
Does this new version work smooth on your device with MK games ? IMO 0.72 would 
run them smooth on 800mhz-1ghz devices as mame4droid 1.5.2.On my old cpu i used 
mame 0.72 for MK games and new versions for games like nightslashers which are 
beatem-ups and all beatem-ups are working nice on reloaded version.CPS1 and 
CPS2 is basically full speed and reloaded is currently best emu to play CPS 
games.
```

Original comment by `2black...@gmail.com` on 28 Feb 2012 at 4:11

**Comment by GoogleCodeExporter** [comment]
```
I have no plans to port 0.72 knowing that next year phones could play 
mame4droid reloaded at very good speed on the few drivers bad optimized. I 
don't have so much time. I prefer to develop the iOS version and add some 
missing stuff on current mame ports. But who knows... :) sadly 0.72 it is a 
very different mame architecture and needs to rewrite the osd layer... 
```

Original comment by `seleuco....@gmail.com` on 28 Feb 2012 at 4:44

**Comment by GoogleCodeExporter** [comment]
```
I see:) if only phones would have better batterries, they keep adding more 
cores but battery still sucks and lasts half of the day, its not cellphone 
anymore but basically small cpu and they should develop some good power source 
lasting much longer than couple of hours.Dual core probably drains battery even 
faster.Great work, my favourites games work nicely.
```

Original comment by `2black...@gmail.com` on 28 Feb 2012 at 4:54

**Comment by GoogleCodeExporter** [comment]
```
The battery drain is related to the screens more than the cpus... Newer cpus 
are developed with fewer layers like new samsung cpus... The drain is about 50 
percentage less than current generation on new exynos

http://www.knowyourmobile.com/blog/1258177/samsung_quad_core_chips_to_be_50_per_
cent_more_energy_efficient.html
```

Original comment by `seleuco....@gmail.com` on 28 Feb 2012 at 5:17

**Comment by GoogleCodeExporter** [comment]
```
yeah im using screen adjuster app to adjust brightness to lower than minimum 
settings to save battery whenever i can but if battery drains so fast and 
everyone complains about it then they should figure out something a couple of 
years ago but were still in the same place and looks like they have to be 
pushed by processor engineers to develop battery and peoples complaints dont 
mean nothing cause they'll buy the phone to be trendy anyway and it doesnt 
matter if they would have to be on power wire all the time to play games.When 
im in home then i play with connected charger.We will see how it will be after 
one year from now:) Personally i would prefere smartphone which would support 
windows 95 just for the sake of all windows softwares,games and emulators, 
Android is cool but its still very fresh OS and theres a lot of software/apps 
but a lot of them are so so.Windows95 is old system but doesnt eat so much 
resources, its possible to run it on the phone from what i saw and it runs 
smooth( http://forum.xda-developers.com/showthread.php?t=1459153 ) but controls 
sucks, obviously keyboard is needed also im not sure how some stuff would work 
not being written for ARM.Its software that makes hardware worthy for me, 
without it its just piece of powerful hardware, but it wont even play oldest 
pacman game without needed software.Now when you ported new MAME version to 
Android i have best portable emu machine i could get at this moment. Thanks :)
```

Original comment by `2black...@gmail.com` on 28 Feb 2012 at 5:54
